profiles/cpu-partitioning/script.sh

```diff
@@ -39,8 +39,7 @@ start() {
     mkdir -p "${TUNED_tmpdir}/usr/lib/dracut/hooks/pre-udev"
     cp /etc/systemd/system.conf "${TUNED_tmpdir}/etc/systemd/"
     cp 00-tuned-pre-udev.sh "${TUNED_tmpdir}/usr/lib/dracut/hooks/pre-udev/"
-    sed -i '/^IRQBALANCE_BANNED_CPUS=/d' /etc/sysconfig/irqbalance
-    echo "IRQBALANCE_BANNED_CPUS=$TUNED_isolated_cpumask" >>/etc/sysconfig/irqbalance
+    irqbalance_banned_cpus_setup $TUNED_isolated_cpumask
```

Comment:
Off the top of my head, I don't know what the format of `$TUNED_isolated_cpumask` is, however I think it would be safer to enclose it in double quotes.